diff_drive_controller/include/diff_drive_controller/diff_drive_controller.h

```diff
@@ -118,24 +118,30 @@ namespace diff_drive_controller
      * \brief Wheel speed limiter setter
      * \param wheel_speed_limiter Wheel speed limiter
      */
-    void setWheelSpeedLimiter(WheelSpeedLimiter wheel_speed_limiter)
```

Comment:
Same for https://github.com/clearpathrobotics/ros_controllers/pull/52/files#diff-1f4682d629140da269af3d94f157326aL132